Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "macro_rules_attribute"
-version = "0.2.1-rc1"  # Keep in sync
+version = "0.2.1-rc2"  # Keep in sync
```

Comment:
Awesome! https://docs.rs/macro_rules_attribute/0.2.1-rc2

Since I might as well not touch the `git` stuff myself, would you mind now doing the `0.2.1` official release?

```bash
./change_version.sh 0.2.1
git commit -pm 'Version 0.2.1 release'
```

That way _your_ commit shall be the one released to crates.io 🙂

Reply:
Done! Thanks again.